src/main/java/mate/academy/spring/validation/PasswordValidator.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.spring.validation;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import mate.academy.spring.dto.request.UserRequestDto;
+
+public class PasswordValidator implements ConstraintValidator<ValidPassword, UserRequestDto> {
+    @Override
+    public boolean isValid(UserRequestDto userRequestDto,
+                           ConstraintValidatorContext constraintValidatorContext) {
+        String plainPassword = userRequestDto.getPassword();
+        String repeatPassword = userRequestDto.getRepeatPassword();
+
+        if (plainPassword == null || plainPassword == "" || repeatPassword == ""
+                || repeatPassword == null || !plainPassword.equals(repeatPassword)) {
```

Comment:
simplify condition and use  return condition; instead of if(condition) return true else return false